test/functional/apps/discover/_discover.js

```diff
@@ -229,6 +231,80 @@ export default function ({ getService, getPageObjects }) {
       });
     });
 
+    describe('query language switching', function () {
+
+      after(async function () {
+        await kibanaServer.uiSettings.replace({
+          'dateFormat:tz':'UTC',
+          'defaultIndex':'logstash-*'
+        });
```

Comment:
How about factoring out the settings document as a constant to use here and in line 22?